### ISSUE TITLE
Missing $libdir/timescaledb-tsl-2.0.x files

### ISSUE THREAD

**Issue by hgiasac** [timescale/timescaledb-docker-ha#85]
I face an issue when upgrading to TimescaleDB 2.0.x. I check that $libdir/timescaledb-tsl-2.0.x.so libraries are missing
These files still exist in the original image `timescale/timescaledb:2.0-pg12` 
I'm not sure if it's expected behavior

```sql
postgres=# ALTER EXTENSION timescaledb UPDATE;                                                                                                                                                 
ERROR:  could not access file "$libdir/timescaledb-tsl-2.0.1": No such file or directory 
```

Thanks

**Comment by feikesteenbergen** [comment]
Could you provide some more details? What version of the built images do you use? Do you build the images yourself?

**Comment by hgiasac** [comment]
Sorry about that. I used `timescaledev/timescaledb-ha/pg12.5-ts2.0.1-latest` version

https://hub.docker.com/layers/timescaledev/timescaledb-ha/pg12.5-ts2.0.1-latest/images/sha256-18973af2ce23dd510caa2c2fef73cd55a8aa8b16ced6e98d55249e55f4c0e500?context=explore

**Comment by feikesteenbergen** [comment]
I'm unable to reproduce the issue, could you provide all the steps you take to run into this issue?
As an example, this is my session:


```console
docker run -d --rm --name bugtest -e POSTGRES_PASSWORD=bugtest timescaledev/timescaledb-ha@sha256:18973af2ce23dd510caa2c2fef73cd55a8aa8b16ced6e98d55249e55f4c0e500
docker exec -ti bugtest psql
```
*psql*
```
psql (12.5 (Debian 12.5-1.pgdg100+1))
Type "help" for help.

postgres=# \dx timescale
     List of installed extensions
 Name | Version | Schema | Description 
------+---------+--------+-------------
(0 rows)

postgres=# \dx timescaledb
                                    List of installed extensions
    Name     | Version | Schema |                            Description                            
-------------+---------+--------+-------------------------------------------------------------------
 timescaledb | 2.0.1   | public | Enables scalable inserts and complex queries for time-series data
(1 row)

postgres=# drop extension timescaledb;
DROP EXTENSION
postgres=# \c
You are now connected to database "postgres" as user "postgres".
postgres=# create extension timescaledb with version "1.7.2";
WARNING:  
WELCOME TO
 _____ _                               _     ____________  
|_   _(_)                             | |    |  _  \ ___ \ 
  | |  _ _ __ ___   ___  ___  ___ __ _| | ___| | | | |_/ / 
  | | | |  _ ` _ \ / _ \/ __|/ __/ _` | |/ _ \ | | | ___ \ 
  | | | | | | | | |  __/\__ \ (_| (_| | |  __/ |/ /| |_/ /
  |_| |_|_| |_| |_|\___||___/\___\__,_|_|\___|___/ \____/
               Running version 1.7.2
For more information on TimescaleDB, please visit the following links:

 1. Getting started: https://docs.timescale.com/getting-started
 2. API reference documentation: https://docs.timescale.com/api
 3. How TimescaleDB is designed: https://docs.timescale.com/introduction/architecture

Note: TimescaleDB collects anonymous reports to better understand and assist our users.
For more information and how to disable, please see our docs https://docs.timescaledb.com/using-timescaledb/telemetry.

CREATE EXTENSION
postgres=# \c
You are now connected to database "postgres" as user "postgres".
postgres=# alter extension timescaledb update to "1.7.3";
ALTER EXTENSION
postgres=# \c
You are now connected to database "postgres" as user "postgres".
postgres=# alter extension timescaledb update to "2.0.1";
ALTER EXTENSION
postgres=# select version from pg_available_extension_versions where name = 'timescaledb' order by version;
  version  
-----------
 1.7.0
 1.7.1
 1.7.2
 1.7.3
 1.7.4
 2.0.0
 2.0.0-rc3
 2.0.0-rc4
 2.0.1
(9 rows)
```

**Comment by hgiasac** [comment]
Sorry for the late reply

I used `v2.0.0-rc3` first, then upgrade to `v2.0.1`
The first issue is Timescale license. `v2.0.0` and newer versions force `timescale` license to enable continuous materialized view. However, `v2.0.0-rc3` use `apache` license by default.

When I updated the license, the error happened. 
```
ALTER SYSTEM SET timescaledb.license = 'timescale';
```

I think this is upgrade issue from v2 beta or RC to stable versions. So it's fine if you create fresh database server with the latest version.
